hypertools/tools/cluster.py

```diff
@@ -1,11 +1,15 @@
 #!/usr/bin/env python
 
 from sklearn.cluster import KMeans
-from sklearn.decomposition import PCA
+import warnings
 import numpy as np
 from .._shared.helpers import *
+from .reduce import reduce as reducer
+from .align import align as aligner
+from .normalize import normalize as normalizer
 
-def cluster(x, n_clusters=8, ndims=None):
+def cluster(x, n_clusters=8, ndims=None, model='IncrementalPCA',
```

Comment:
sounds reasonable - we also need a way to specify model parameters. what about..

`y = hyp.plot(x, reduce='TSNE', reduce_params={'sparsity': 40}, cluster='kmeans', align='SRM', ndims=5)`

similarly, `cluster_params` and `align_params` would allow the user to pass params to the cluster and align models, respectively.

We probably want to keep `model` and `model_params` but if they are used, we could create a warning that will say that these arguments will be deprecated in a future version. or we could just continue to support them